test/paper_submenu_test.dart

```diff
@@ -0,0 +1,169 @@
+@TestOn('browser')
+library polymer_elements.test.paper_progress_test;
+
+import 'dart:html';
+import 'package:polymer_elements/paper_item.dart';
+import 'package:polymer_elements/paper_submenu.dart';
+import 'package:polymer_elements/paper_menu.dart';
+import 'package:polymer_elements/iron_collapse.dart';
+import 'package:web_components/web_components.dart';
+import 'package:test/test.dart';
+import 'common.dart';
+
+/**
+ * Original tests:
+ * https://github.com/PolymerElements/paper-menu/tree/master/test
+ */
+
+main() async {
+  await initWebComponents();
+
+  group('<paper-submenu>', () {
+    PaperMenu menu;
+    PaperSubmenu sub1, sub2, sub3;
+    IronCollapse collapse1, collapse2, collapse3;
+    PaperItem trigger1, trigger2, trigger3;
+
+    setUp(() {
+      menu = fixture('basic');
+
+      sub1 = menu.querySelectorAll('paper-submenu')[0];
+      sub2 = menu.querySelectorAll('paper-submenu')[1];
+      sub3 = menu.querySelectorAll('paper-submenu')[2];
+
+      collapse1 = sub1.querySelector('iron-collapse');
+      collapse2 = sub2.querySelector('iron-collapse');
+      collapse3 = sub3.querySelector('iron-collapse');
+
+      trigger1 = sub1.querySelector('.menu-trigger');
+      trigger2 = sub2.querySelector('.menu-trigger');
+      trigger3 = sub3.querySelector('.menu-trigger');
+    });
+
+    test('selecting an item expands the submenu', () {
+      expect(collapse1.opened, isFalse);
+      expect(collapse2.opened, isFalse);
+      expect(collapse3.opened, isFalse);
+
+      tap(trigger1);
+
+      expect(collapse1.opened, isTrue);
+      expect(collapse2.opened, isFalse);
+      expect(collapse3.opened, isFalse);
+    });
+
+    test('selecting a different item closes the previously opened submenu', () {
+      expect(collapse1.opened, isFalse);
+      expect(collapse2.opened, isFalse);
+      expect(collapse3.opened, isFalse);
+
+      tap(trigger1);
+
+      expect(collapse1.opened, isTrue);
+      expect(collapse2.opened, isFalse);
+      expect(collapse3.opened, isFalse);
+
+      tap(trigger2);
+
+      expect(collapse1.opened, isFalse);
+      expect(collapse2.opened, isTrue);
+      expect(collapse3.opened, isFalse);
+    }, skip: 'https://github.com/dart-lang/polymer_elements/issues/30');
```

Comment:
The behaviors should work, you just can't mix them into a polymer dart element (actually, they do work in [this branch](https://github.com/dart-lang/polymer_elements/tree/behaviors)). I would make a new issue, but one overall one for all tests in this element is probably ok.